hogben/simulate.py

```diff
@@ -268,7 +289,9 @@ def reflectivity(q, model):
             experiment = refl1d_experiment(model.sample, q, scale, bkg, dq)
             return experiment.reflectivity()[1]
 
-def refl1d_experiment(sample, q_array, scale, bkg, dq, spin_state=None):
+
+def refl1d_experiment(sample, q_array: np.ndarray, scale: float, bkg: float, dq: float,
```

Comment:
Same comment on types no longer being needed in the docstring, but descriptions of np array types/shapes are useful